cli/dcoscli/analytics.py

```diff
@@ -261,6 +261,12 @@ def _base_properties(conf=None):
         logger.exception('Unable to find the hostname of the cluster.')
         dcos_hostname = None
 
+    try:
+        cluster_id = mesos.DCOSClient().metadata().get('CLUSTER_ID')
```

Comment:
@tamarrow Can you confirm that we only make this call when `core.reporting` is `True`. In other words we shouldn't make this call when reporting is disabled.